vault/overlays/nerc-ocp-infra/backup-job/task.yaml

```
@@ -0,0 +1,99 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: backup-vault
+spec:
+  workspaces:
+    - name: snapshots
+      mountPath: /snapshots
+    - name: backup-job
+      mountPath: /backup-job
+  steps:
+    - name: create-snapshot
+      image: quay.io/operate-first/opf-toolbox:v0.8.0
+      workingDir: /workspace
+      script: |
+        #!/bin/sh
+        set -e
+
+        echo 'Logging into nerc-vault-0 to retrieve leader'
+        oc exec -i nerc-vault-0 -- vault operator raft list-peers --format=json > /tmp/peers.json
+        leader=$(yq e '.data.config.servers.[] | select(.leader == "true") | .address' -P /tmp/peers.json | cut -d '.' -f1)
+
+        echo retrieved leader: ${leader}
+        echo Creating Snapshot in leader pod ${leader}...
+
+        export snapshot_file_path=$(echo snapshot-"`date +"%F_T%H-%M-%S"`".snap)
```

There's no need for `echo` here (or for nested command substitution):

```suggestion
        export snapshot_file_path=$(date +"snapshot-%F_T%H-%M-%S.snap")
```